src/core/Global.ttslua

```diff
@@ -39,6 +39,12 @@ local mythosAreaApi = require("core/MythosAreaApi")
 local tokenArrangerApi = require("accessories/TokenArrangerApi")
 local blessCurseManagerApi = require("chaosbag/BlessCurseManagerApi")
 
+-- online functionality related variables
+local MOD_VERSION = "3.1.0"
+local SOURCE_REPO = 'https://raw.githubusercontent.com/chr1z93/loadable-objects/main'
+local library, requestObj, modMeta, notificationVisible
+local dontNotify = {}
```

Comment:
Prefer positive logic here rather than negative (which is often confusing).  acknowledgedUpgradeVersions avoids the negative and also explains what is stored in the list